test/python/compiler/test_transpiler.py

```diff
@@ -1981,12 +1961,12 @@ def run(self, dag):
                 )
                 return dag
 
-        backend = FakeMumbaiV2()
+        backend = FakeGeneric(num_qubits=4)
 
         # This target has PulseQobj entries that provides a serialized schedule data
         pass_ = TestAddCalibration(backend.target)
         pm = PassManager(passes=[pass_])
-        self.assertIsNone(backend.target["sx"][(0,)]._calibration._definition)
+        # self.assertIsNone(backend.target["sx"][(0,)]._calibration._definition)
```

Comment:
This is the only test( just this single line ) that is failing, that is why I have commented it out. I am sure I am missing something in the FakeGeneric.
backend.target['sx'][(0,)]._calibration returns None in FakeGeneric where as for FakeMumbaiV2 it returns a calibration.
That is why backend.target["sx"][(0,)]._calibration._definition errors out.